src/zampy/datasets/land_cover.py

```diff
@@ -0,0 +1,249 @@
+        )
+        target_dataset = xarray_regrid.create_regridding_dataset(new_grid)
+
+        ds_regrid = ds.regrid.most_common(target_dataset, time_dim="time", max_mem=1e9)
+
+        return ds_regrid
+
+    def convert(
+        self,
+        ingest_dir: Path,
+        convention: str | Path,
+    ) -> bool:
+        converter.check_convention(convention)
+        ingest_folder = ingest_dir / self.name
+
+        data_file_pattern = "land-cover_LCCS_MAP_*.nc"
+
+        data_files = list(ingest_folder.glob(data_file_pattern))
+
+        for file in data_files:
+            # start conversion process
+            print(f"Start processing file `{file.name}`.")
+            ds = xr.open_dataset(file)
+            ds = converter.convert(ds, dataset=self, convention=convention)
+            # TODO: support derived variables
+            # TODO: other calculations
+            # call ds.compute()
+
+        return True
+
+
+def unzip_raw_to_netcdf(
+    ingest_folder: Path,
+    file: Path,
+    overwrite: bool = False,
+) -> None:
+    """Convert a downloaded zip netcdf file to a standard CF/Zampy netCDF file.
+
+    Args:
+        ingest_folder: Folder where the files have to be written to.
+        file: Path to the land cover .zip archive.
+        overwrite: Overwrite all existing files. If False, file that already exist will
+            be skipped.
+    """
+    ncfile = ingest_folder / file.with_suffix(".nc").name
+    if ncfile.exists() and not overwrite:
+        print(f"File '{ncfile.name}' already exists, skipping...")
+    else:
+        ds = extract_netcdf_to_zampy(file)
+        ds.to_netcdf(path=ncfile)
+
+
+def extract_netcdf_to_zampy(file: Path) -> xr.Dataset:
+    """Extract zipped data and convert to zampy format.
+
+    Args:
+        file: Path to the land cover .zip archive.
+
+    Returns:
+        Coarse land cover data in zampy format.
+    """
+    with TemporaryDirectory(ignore_cleanup_errors=True) as temp_dir:
+        unzip_folder = Path(temp_dir)
+        with ZipFile(file, "r") as zip_object:
+            zipped_file_name = zip_object.namelist()[0]
+            zip_object.extract(zipped_file_name, path=unzip_folder)
+
+        # only keep land cover class variable
+        with xr.open_dataset(unzip_folder / zipped_file_name) as ds:
+            var_list = [var for var in ds.data_vars]
+            raw_variable = "lccs_class"
+            var_list.remove(raw_variable)
+            ds = ds.drop_vars(var_list)  # noqa: PLW2901
+
+            # coarsen to fit into memory
+            ds = ds.sortby(["lat", "lon"])  # noqa: PLW2901
+            ds = ds.rename({"lat": "latitude", "lon": "longitude"})  # noqa: PLW2901
+            new_grid = xarray_regrid.Grid(
+                north=90,
+                east=180,
+                south=-90,
+                west=-180,
+                resolution_lat=0.25,  # same as resolution of ERA5, must be sufficient
+                resolution_lon=0.25,
```

Comment:
let's mention the default values of the resolutions in the docstring.

Reply:
Thanks for the notice! I will include the choice in the docstrings.

Reply:
> Perhaps it might even be better to expose this somehow... I'm not sure if the ERA5 resolution will always be sufficient. Not sure of a nice way of doing this though :/

It seems there is quite some discussion about which parameters should be configurable. It is easy to add it to the functions, but they do not align nicely with the recipes. 😅 Let me create an issue for these configurable choices then.